erpnext/selling/doctype/sales_order/sales_order_list.js

```diff
@@ -15,18 +15,24 @@ frappe.listview_settings['Sales Order'] = {
 			// not delivered & overdue
 				return [__("Overdue"), "red",
 					"per_delivered,<,100|delivery_date,<,Today|status,!=,Closed"];
+			} else if (doc.docstatus == 1 && doc.per_billed == 100 && doc.per_delivered < 100) {
```

Comment:
Change the `per_delivered` to `per_picked`. The `per_delivered` is already happening on line 13. Also do this on line 21.